### PR TITLE
[ADD] condominium: add the meter readings

### PR DESCRIPTION
**Pull request by dhrs-odoo**
task-4311819

### CODE REVIEW

**Review by vava-odoo — COMMENTED** [review]
Just a few comments, thanks!

File: condominium/data/meter_view.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="meters_list_view" model="ir.ui.view">
```
[the remark this answers is not in the document]
Comment (dhrs-odoo) [reply condominium/data/meter_view.xml:3]: Bcz if we keep this in ir.ui.view then we need to move that first before ir_actions_act_window which will cause error as we have used those actions in the view file

[the remark this answers is not in the document]
Comment (vava-odoo) [reply condominium/data/meter_view.xml:3]: It makes sense, indeed

**Comment by dhrs-odoo** [comment]
Hello @vava-odoo 
I have applied all the suggestions
Please have a look at it 
Thank You :innocent:

**Review by vava-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**